WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -355,6 +357,14 @@ import Combine
         } else if (siteID != nil || tagSlug != nil) && isShowingResultStatusView == false {
             displayLoadingStream()
         }
+
+        if let readerTopic {
+            readerTopic.objectWillChange
+                .sink { [weak self] _ in
+                    self?.updateStreamHeaderIfNeeded()
+                }
+                .store(in: &cancellables)
+        }
```

Comment:
Did you consider optional chaining?

```suggestion
        readerTopic?
            .objectWillChange
            .sink { [weak self] _ in
                self?.updateStreamHeaderIfNeeded()
            }
            .store(in: &cancellables)
```

I tried it locally and it worked. 🤷‍♂️

Reply:
I did not 😓 . Neat idea though, I've created a follow up PR https://github.com/wordpress-mobile/WordPress-iOS/pull/20171

Reply:
Thank you for considering my suggestion.